modules/claims_api/spec/support/swagger_shared_components/v2.rb

```diff
@@ -122,7 +122,9 @@ def self.body_examples # rubocop:disable Metrics/MethodLength
           schema: {
             type: :object,
             required: disability_compensation_json_schema['required'],
-            properties: disability_compensation_json_schema['properties'],
+            properties: {
+              attributes: disability_compensation_json_schema
+            },
```

Comment:
Intent-to-file does not have this change, and has the desired data and attributes. Still working out how...
![Screenshot 2023-05-10 at 1 26 43 PM](https://github.com/department-of-veterans-affairs/vets-api/assets/25069483/ede9adc1-0da6-4a2d-9336-7b950427306a)

Reply:
Visually it can be achieved in at least two ways, by adding it in directly in the 526.json schema file or by adding it in the v2.rb file.  When I changed the 526.json directly to have that stuff nested I got a bunch of failed tests because it could not find the nested fields, it said they were not defined in the schema, even though they were.  Since it was breaking tests for me I assumed that was not the correct way to proceed and so I looked at how v1 achieved that.   It works for intent to file there because it is directly nested in the schema file which links to another json file and appears to parse out the sections 
![Screenshot 2023-05-10 at 2 45 14 PM](https://github.com/department-of-veterans-affairs/vets-api/assets/129893414/21fdf0c8-54e7-43f0-bd32-b4cc42c5b266)

I am totally fine with doing this however you think the correct way forward on this should be.  Embedding it right in the schema file, the v2 config (which I assume is not wanted) or even just linking to the file like they do here.